src/EnergyPlus/Psychrometrics.hh

```diff
@@ -129,7 +129,405 @@ namespace Psychrometrics {
                                 // 14 - HR | 15 - max iter | 16 - HR | 17 - max iter | 18 -
                                 // PsyTwbFnTdbWPb_raw (raw calc) | 19 - PsyPsatFnTemp_raw
                                 // (raw calc)
-
+// sample bin size =64 Pa; sample size =1651 (continous)
+     constexpr std::array<Real64, 1651> tsat_fn_pb_x = {
```

Comment:
I don't think this array is necessary since the contents are just index * 64, right?

Reply:
Good point, the array is just 64x an integer so we can probably either populate it with a for loop or eliminate it.

Reply:
There is no point in populating it with a loop. If you are going to do that, you may as well leave it as is. Just replace the lookup with a multiplication.

Reply:
@amirroth @JasonGlazer  Thanks a lot for your comments! I will update the code to eliminate the pressure input array and replace it with a multiplication.